common/djangoapps/enrollment/management/commands/enroll_user_in_course.py

```diff
@@ -0,0 +1,56 @@
+"""
+Management command for enrolling a user into a course via the enrollment api
+"""
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from enrollment.api import add_enrollment
+
+
+class Command(BaseCommand):
+    """
+    Enroll a user into a course
+    """
+    help = """
+    This enrolls a user into a given course as "audit", "verified" or "honor". Email is required.
+
+    example:
+        # Enroll a user test@example.com into the demo course
+        manage.py ... enroll_user_in_course -e test@example.com -c edX/Open_DemoX/edx_demo_course -m verified
+
+        If a user is already enrolled, the command will return an error.
```

Comment:
Because the goal is to be able to use this command from ansible, do we want it to be idempotent? Maybe not return an error but instead succeed if the user is already enrolled in the course?